src/style.css

```diff
@@ -1 +1,124 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  color: #3D4E61;
+}
+
+/* #region search-bar */
```

Comment:
```suggestion
```

Reply:
not fixed
<img width="857" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64750958/7680b094-978f-4e73-8e65-8c41aead1788">